paywall/src/config.js

```diff
@@ -68,6 +68,9 @@ export default function configure(
 
   const env = runtimeConfig.unlockEnv
 
+  let paywallUrl = runtimeConfig.paywallUrl || 'http://127.0.0.1:3001'
```

Comment:
note: the choice of `127.0.0.1:3001` is very intentional. This will trigger cross-domain protections when accessed from `localhost:3001`, which is where the demo will live.

Reply:
Could you please add that as a comment above that line? I worry that we will forget the reason for this in the future..

Reply:
good idea!